scripts/bump-chrome-driver-version.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+const request = require('request')
+
+const {
+  printLog,
+  printError,
+  logAndExit,
+  executeCommand,
+  replaceCiVariable,
+  initGitConfig,
+  getSecretKey,
+} = require('./utils')
+
+const CI_FILE = '.gitlab-ci.yml'
+const REPOSITORY = process.env.GIT_REPOSITORY
+const MAIN_BRANCH = process.env.MAIN_BRANCH
+const CURRENT_CI_IMAGE = process.env.CURRENT_CI_IMAGE
+
+const CURRENT_PACKAGE_VERSION = process.env.CHROME_PACKAGE_VERSION
+const CHROME_PACKAGE_URL = 'https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-stable'
+const CHROME_DRIVER_URL = 'https://chromedriver.storage.googleapis.com/?delimiter=/&prefix='
+
+async function main() {
+  await initGitConfig(REPOSITORY)
+  await executeCommand(`git fetch --no-tags origin ${MAIN_BRANCH}`)
+  await executeCommand(`git checkout ${MAIN_BRANCH} -f`)
+  await executeCommand(`git pull origin ${MAIN_BRANCH}`)
+
+  const packageVersion = await getPackageVersion()
+
+  if (packageVersion <= CURRENT_PACKAGE_VERSION) {
```

Comment:
By doing an alphabetical comparison, we might have an issue when the version will reach `100`. It could be manually updated when we face this issue.

Reply:
I fixed the check to only compare major version number (as numbers)